roles/lib_utils/library/oo_azure_rm_publish_image_facts.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/env python
```

Comment:
oo_azure_rm_publish_image_facts.py and oo_azure_rm_publish_image.py are very similar indeed - on what basis are there two files?

Reply:
The ansible pattern is to have a `_facts` version that does the reading / list.  The other does the actual heavy lifting.